src/handlers/next/saleor-webhooks/sync-webhook-response-builder.ts

```diff
@@ -0,0 +1,84 @@
+import { SyncWebhookEventType } from "../../../types";
```

Comment:
// just a note

Not sure if you are writing types by hand, but for such operations I like to use https://quicktype.io/typescript to speed up the boring part ;) 

Wonder if there are any sync interfaces exposed by the core, we could codegen from OpenAPI schemas 🤔

Reply:
Nice! I usually replace with regexp in IDE but this tool is great

>Wonder if there are any sync interfaces exposed by the core, we could codegen from OpenAPI schemas 🤔

I asked @przlada to verify how core represents it. Im afraid we don't have such thing, but we already confirmed with Patrys that we should have some shared contracts between core & apps for AppManifest, payloads etc (everything non graphql basically). But it requires some further discussions, there are plenty ways to achieve it